### PR TITLE
Mo 81 add successful notification for user when he filled working hours in timesheet

### PR DESCRIPTION
**Pull request by RajatMCAF**
[ADD] added the notification box

When correct value passes through cell

### CODE REVIEW

File: mo_hr_timesheets/static/src/js/grid_controller.js [hunk 1]
```diff
@@ -1,18 +1,60 @@
...
+    var core = require('web.core');
+    var dialogs = require('web.view_dialogs');
+    var utils = require('web.utils');
+    var concurrency = require('web.concurrency');
+    var qweb = core.qweb;
+    var _t = core._t;
 
     GridController.include({
-        _grid_adjust: function (cell, newValue) {
+        OnTestClicks: function() {
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/static/src/js/grid_controller.js:17]: rename function

File: mo_hr_timesheets/static/src/js/grid_controller.js [hunk 2]
```diff
@@ -1,18 +1,60 @@
...
+        },
+
+        _onCellEdited: function(event) {
+            var state = this.model.get();
+            this._grid_adjust({
+                row: utils.into(state, event.data.row_path),
+                col: utils.into(state, event.data.col_path),
+                value: utils.into(state, event.data.cell_path).value,
+            }, event.data.value).then(() => {
+                this.OnTestClicks();
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/static/src/js/grid_controller.js:38]: rename function

File: mo_hr_timesheets/static/src/js/grid_controller.js [hunk 3]
```diff
@@ -1,18 +1,60 @@
...
+    var GridView = require('generic_grid.GridView');
+    var registry = require('@web/core/registry');
+    var AbstractController = require('web.AbstractController');
+    var config = require('web.config');
+    var core = require('web.core');
+    var dialogs = require('web.view_dialogs');
+    var utils = require('web.utils');
+    var concurrency = require('web.concurrency');
+    var qweb = core.qweb;
+    var _t = core._t;
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/static/src/js/grid_controller.js:15]: add documentation what the method does

File: mo_hr_timesheets/static/src/js/grid_controller.js (hunk 1 above)
Comment (RajatMCAF) [reply mo_hr_timesheets/static/src/js/grid_controller.js:17]: refracted to notifyHoursPosted

File: mo_hr_timesheets/static/src/js/grid_controller.js (hunk 2 above)
Comment (RajatMCAF) [reply mo_hr_timesheets/static/src/js/grid_controller.js:38]: refracted to notifyHoursPosted

File: mo_hr_timesheets/static/src/js/grid_controller.js (hunk 3 above)
Comment (RajatMCAF) [reply mo_hr_timesheets/static/src/js/grid_controller.js:15]: Done

**Review by MORPSmcaf — APPROVED** [review]

**Pull request merged**